node-definitions/cisco/ise/ise.yaml

```diff
@@ -50,4 +50,51 @@ inherited:
 configuration:
   generator:
     driver: null
+  provisioning:
+    files:
+      - editable: true
+        name: ise-ztp.conf
+        content: |-
+          hostname=<hostname of Cisco ISE>
```

Comment:
@BregaladTaran Can you fill some of these in with defaults?  I'm thinking hostname, admin password, and domain.